<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>182</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="btnNext">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>140</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>next</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnPlay">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>140</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>play</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnPrev">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>prev</string>
       </property>
      </widget>
      <widget class="QListWidget" name="lstScreen">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>0</y>
         <width>381</width>
         <height>131</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btnPause">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>140</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Pause</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
